app.js

```diff
@@ -0,0 +1,87 @@
+var express = require('express')
+var http = require('http')
+var path = require('path')
+var passport = require('passport')
+var LocalStrategy = require('passport-local').Strategy
+
+//==================================================================
+// Define the strategy to be used by PassportJS
+passport.use(new LocalStrategy(
+  function(username, password, done) {
+    // @todo Tie this authentication to the members database in CouchDB
+    if (username === "admin" && password === "admin") // stupid example
+      return done(null, {name: "admin"})
+
+    return done(null, false, { message: 'Incorrect username.' })
+  }
+))
+
+// Serialized and deserialized methods when got from session
+passport.serializeUser(function(user, done) {
+    done(null, user)
+})
+
+passport.deserializeUser(function(user, done) {
+    done(null, user)
+})
+
+//==================================================================
+// Start express application
+var app = express()
+
+// all environments
+app.set('port', process.env.PORT || 3000)
+app.use(express.favicon())
+app.use(express.logger('dev'))
+app.use(express.cookieParser()) 
+app.use(express.bodyParser())
+app.use(express.methodOverride())
+app.use(express.session({ secret: 'securedsession' }))
+app.use(passport.initialize()) // Add passport initialization
+app.use(passport.session())    // Add passport initialization
+app.use(app.router)
+app.use(express.static(path.join(__dirname, 'public')))
```

Comment:
This is the line that allows the `public` directory to be mounted and available at the root of the URL. In other words, the BeLL Apps could now be accessed at something like `http://bell.local/MyApp/index.html`. Or even better, we could create an `public/index.html` file that redirects to MyApp or displays some links as the "front door" of the server for users.